apps/sms/services/js/messaging/messaging_server.js

```diff
@@ -0,0 +1,542 @@
+/* -*- Mode: js; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- /
```

Comment:
All the implementation was borrowed from message manager and we might be able to reuse these code in follow up.